language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/CodegenLedgerTest.scala

```diff
@@ -83,8 +83,8 @@ class CodegenLedgerTest
       sruquitoContract.data shouldEqual sruquito
 
       val tob = Instant.now().`with`(ChronoField.NANO_OF_SECOND, 0)
-      val reproduceCmd = glookoflyContract.id
-        .exerciseReproduce(sruquitoContract.id, tob)
+      val update = glookoflyContract.id.exerciseReproduce(sruquitoContract.id, tob)
+      val reproduceCmd = update.command()
```

Comment:
I would revert this (and all similar changes in this file) and define overload[s] for `sendCmd` that accepts an `Update[_]` as its third argument, instead. Or a typeclass that unifies `Command` and `Update` if we expect to still have tests that want to pass `Command` but don't have an `Update` to pass.